src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -58,6 +58,20 @@ public DebugService(PowerShellContext powerShellContext)
         #region Public Methods
 
         /// <summary>
+        /// Disables all breakpoints in the runspace.
+        /// </summary>
+        /// <returns></returns>
+        public async Task DisableAllBreakpoints()
```

Comment:
Here's a possible strategy.  Let's go ahead and check in a solution where we just assume the message ordering comes in correctly and that we know the value of NoDebug before the breakpoint request comes in.  If NoDebug is on, we'll just ignore any breakpoint request that comes through.

Since the 1.0 release of VS Code is supposed to land at the end of April, we can keep an eye on it and see if the debug messages continue to be sent in the right order.  We'd then release our update on or just after the 1.0 release is done.  If it turns out people see issues with running in NoDebug mode, we can release a quick bugfix update that re-introduces the breakpoint disabling code.

I think I'd rather take this strategy than have to make further changes to our code to avoid a possible message ordering problem.  What do you think?